README.md

```diff
@@ -1,50 +1,73 @@
 # QDD
+
 ## About
+
 QDD is a decision diagram based quantum computing simulator.
 Memory usage can be significantly reduced compared to typical state vector based simulators.
 
 It works as a Qiskit backend.
 At this point, we do not distribute wheel file, so you need to build it locally.
 
 ## License
+
 BSD 3-Clause Clear License
 
 ## Install
+
 Supported environment
-* Linux x86_64 with Python3.8 ~ 3.10
+
+- Linux x86_64 with Python3.8 ~ 3.10
 
 Command
+
 ```sh
 $ pip install qdd
 ```
 
 ## Install from source
+
 ### Build
+
 You must prepare the following software.
 
-* cmake (>=3.25)
-  * You can download the executable from [Official GitHub](https://github.com/Kitware/CMake/releases).
-  * If you use Ubuntu, [the Kitware repository](https://apt.kitware.com/) is easier to install.
-* build
-  * Just install it via pip.
+- cmake (>=3.25)
+  - You can download the executable from [Official GitHub](https://github.com/Kitware/CMake/releases).
+  - If you use Ubuntu, [the Kitware repository](https://apt.kitware.com/) is easier to install.
+- build
+  - Just install it via pip.
+
+You can manually build QDD as follows.
 
-You can build QDD as follows.
 ```sh
 $ cmake . -DCMAKE_BUILD_TYPE=Release
 $ cmake --build . -j
 $ python3 -m pip install build
 $ python3 -m build
 ```
+
 BUILD_TYPE can be either `Release`, `Debug` or `RelWithDebInfo`.
 You can find an installable wheel file in 'dist' directory.
 
+Alternatively, you can just execute the shell script in the project root to build it in one line:
+
+```sh
+./scripts/local_build_and_test.sh build`
+```
+
+If virtualenv is not used, it will create a new one named "venv-qdd" under the project root.
+BUILD_TYPE is set to "Release" this way.
+If you want to just build C extension module (.so file), skipping distribution building process, you can specify "cmake" as the first argument of the shell script.
```

Comment:
venv-qdd looks nicer, same for `Release` and `cmake`.
In my opinion the variable names and arguments should be surrounded by "`" character.